tests/openbis_functions/test_interbis.py

```diff
@@ -164,7 +210,7 @@ def test_get_metadata_import_template(setup, pytestconfig, expected_df_import, w
         pd.testing.assert_frame_equal(df, expected_df_import)
 
 
-@pytest.mark.login
+@ pytest.mark.login
```

Comment:
same here, without space?

Reply:
at least do it consistently for the whole file

Reply:
Same as above